### ISSUE TITLE
Add carthage support

### ISSUE THREAD

**Issue by orschaef** [idnow/de.idnow.ios#31]
It would be nice, if you could add Carthage support for iOS. Not everyone in the iOS community uses cocoapods.

Integration in the `Cartfile` should be like this:

`github "idnow/de.idnow.ios"`

or this:

`binary "https://domain.to.frameworksJson/id_now_versions.json"`

Or do I miss something and this is already possible somehow?

**Comment by pvegh** [comment]
No reply within 23 days. Is this project abandoned?

**Comment by orschaef** [comment]
Commits 14 days ago. Seems to be active...

**Comment by andreaskofer** [comment]
Hello,

we will look into your request and let you know.

Best regards,
Andreas

**Comment by orschaef** [comment]
Hi Andreas,

thanks for your reply.

If you need any support I‘m available for help. Also if you need more information for the workflow with carthage or why supporting carthage would be very helpful.

Cheers 🍻
Orlando

**Comment by andreaskofer** [comment]
Hello,

we looked into supporting Carthage. Right now we can not support two different dependency management systems at the same time (we can not drop Cocoapods in favour of Carthage as existing integrations of our SDK rely on that one).

We will put an action item into our backlog to revisit this decision.

Best regards,
Andreas

**Comment by orschaef** [comment]
Sad to hear that. What are the issues you are encountering? From my perspective this shouldn't be much work. Since we really need to integrate your SDK it would be nice if we could collaborate on this problem. You must understand: We don't want to change our architecture setup and dependency management because of a library that brings in a feature that our users will play through only once.

Regards
Orlando

**Comment by martindaum** [comment]
I agree with Orlando, it should definitely be possible to support Cocoapods and Carthage. Both of them are quite popular in the community and should be supported by a commercial framework.

Cheers,
Martin

**Comment by javierHolvi** [comment]
Yeah same here @martindaum ... It will be great to have Carthage support.

**Comment by hixfield** [comment]
Also here, we are looking for an an ID facilitator but Carthage is a must. It really should not be much effort to implement. Any update?

**Comment by orschaef** [comment]
Sadly this seems to be ignored.

**Comment by tjanela** [comment]
We are evaluating the SDK and although a manual integration, compatible with carthage, is possible it is less preferable than this lib supporting carthage from the get-go.

If I have the time to do it I'll provide you guys with a PR.

**Comment by tjanela** [comment]
To whoever ends up here, 
take a look at:
#74 
As it contains an update to the list of dependencies and linker flags.

**Comment by Beniamiiin** [comment]
Guys, I did [it](https://github.com/idnow/de.idnow.ios/pull/87). We just have to wait till they approve it.
